.changeset/sweet-tires-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@api3/contracts': major
```

Comment:
Even though they are unused, I decided to do a major version anyway. There are some Api3 projects for which this will cause breakage.